test/parallel/test-util-format.js

```diff
@@ -127,6 +130,8 @@ assert.strictEqual(util.format('%s', null), 'null');
 assert.strictEqual(util.format('%s', 'foo'), 'foo');
 assert.strictEqual(util.format('%s', 42), '42');
 assert.strictEqual(util.format('%s', '42'), '42');
+assert.strictEqual(util.format('%s', -0), '-0');
+assert.strictEqual(util.format('%s', '-0.0'), '-0.0');
```

Comment:
Worth including assertions for `-0` and `-0.0` too? (That is, numbers rather than strings.)

Reply:
The line above (133) tests for the number `-0`. That should be sufficient, right?

Reply:
Sorry, missed that it didn't have quotation marks around the value. Yes, that's sufficient.